modules/ve-cx/ui/ve.ui.CXLinkContextItem.js

```
@@ -175,8 +175,11 @@ ve.ui.CXLinkContextItem.prototype.renderBody = function () {
  * @return {jQuery.Promise}
  */
 ve.ui.CXLinkContextItem.prototype.getLinkInfo = function () {
+	var thisLanguage = this.inTargetSurface ? this.translation.getTargetLanguage() : this.translation.getSourceLanguage();
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)